sdk/examples/how_tos/simple_transaction/request_funds.rs

```diff
@@ -3,7 +3,7 @@
 
 //! TODO: Example description
 //!
-//! `cargo run --example request_funds --release`
+//! `cargo run --example request_funds --release --features="rocksdb stronghold"`
```

Comment:
```suggestion
//! `cargo run --release --all-features --example request_funds
```
doesn't create the smallest binary, but I find it more convenient and re-usable

Also I don't quite get this example. Don't we already have `get_funds`?

Reply:
I will do whatever you decide. So should I go for --all-features in all examples?

Hmm. Maybe I just missed get_funds 🤔 . Can delete that one I guess. Wdyt?

Reply:
Yes please delete that one so we have not two examples doing the same

Reply:
Deleted and updated the example command